src/appendices/qmri.md

```diff
@@ -136,8 +136,8 @@ Please visit the [file collections appendix](./file-collections.md#magnetic-reso
 ### Outputs are quantitative maps
 
 qMRI maps are stored differently depending on the process that generated them.
-Pre-generated qMRI maps MUST be stored as part of raw BIDS datasets,
-where as it MUST be stored in a derivative BIDS dataset if it were post-generated.
+Pre-generated qMRI maps MAY be stored as part of a raw BIDS dataset,
+where as it MAY be stored in a derivative BIDS dataset if it were post-generated.
```

Comment:
I think this one should be a MUST, even though it's unenforceable. Here the MUST would be best interpreted as a directive to a tool that does the calculation that it should not modify the original dataset.

```suggestion
where as they MUST be stored in a derivative BIDS dataset if they were post-generated.
```

Also, updating pronouns for agreement with "maps".